.travis.yml

```diff
@@ -89,6 +89,14 @@ script:
   - if [ "$SAFETY_PROFILE" != "1" ]; then
       $DDS_ROOT/tests/cmake_integration/run_ci_tests.pl;
     fi
+  - if [ "$NO_TESTS" == "1" ]; then
+      echo Building Messenger;
+      source setenv.sh;
```

Comment:
I thought it already had the environment set

Reply:
Without it it gives an error: "mwc.pl: command not found"

Reply:
Right, it has some environment variables but not exactly the same as setenv.sh.  There's no need for it to be on the PATH when you know where to find it (`$ACE_ROOT/bin/mwc.pl`).

Reply:
Fixed.